website/src/components/CardLevel.vue

```diff
@@ -96,6 +97,10 @@ export default {
       return this.item.hidden === true
     },
 
+    isVerifyQueue() {
+      return this.listType == 'tab_verify_queue';
+    },
+
```

Comment:
Having the list type here now is great, but please just move this check into line 150 instead instead of going through this helper method. Would be overkill to have need one for each different type if we ever were to make more things type dependent.